js/model3D/shared/Model3D.svelte

```diff
@@ -3,6 +3,7 @@
 	import { BlockLabel, IconButton } from "@gradio/atoms";
 	import { File, Download, Undo } from "@gradio/icons";
 	import Canvas3D from "./Canvas3D.svelte";
+	import Canvas3DGS from "./Canvas3DGS.svelte";
```

Comment:
We should dynamically import these components only when necessary. They both bring significant dependencies with them.

Reply:
I'm not sure how to do this, would appreciate help with this if the gradio team decided to go through with this PR.

Reply:
This will probably give some type errors atm but I'd need to check exactly what they are, dynamic imports are a bit weird to type.

Reply:
We could abstract this with a fancy function but it probably isn't worth it atm, the duplication is fine. I should create a utility to make this simpler, we do a lot of dynamic importing.

Reply:
Thanks, I have revised it to use dynamic imports. I had to make use of `any` which I usually avoid in typescript.